### ISSUE TITLE
Crashes when testing

### ISSUE THREAD

**Issue by F2UME** [yzyio/adidas-multi-session#8]
When running the program, is gets through the tests of the proxies you put in place, but then once it completed through a proxy of mine, it's as if it's found a HMAC cookie and then start to spam open chrome windows, which then freezes up my PC causing me to force restart.
![1vscmuv](https://cloud.githubusercontent.com/assets/25882199/23103436/314cfb30-f6b3-11e6-9a90-0ab93cb51a5b.png)

I've tried different proxies etc also.
![2017-02-19_14-55-39](https://cloud.githubusercontent.com/assets/25882199/23103512/056af8fe-f6b4-11e6-980c-63beca9025d1.png)

It seems to be spamming the transfer of session.

_(Also unrelated, but this will work for United Kingdom correct)_

Thanks.

**Comment by CoolA1d** [comment]
A temporary solution would be to add time.sleep(600) right after `threading.Thread(target=transfer_session, kwargs={'browser': browser}).start()` 
in bot.py.

The reason this is happening is because of improper opened browser checking, I'm sure bommels is aware of this and will fix it once v0.2 is fully released.

Also keep in mind you should only be using free test proxies to test because most of the free proxies are blocked by adidas and other sites.

**Comment by F2UME** [comment]
Thanks for that @gt853 Will do, just testing and seeing everything :).

**Comment by F2UME** [comment]
Ok, i've tried that, and the same happened again, I managed to take another screenshot before the freezing.
![2017-02-19_16-55-35](https://cloud.githubusercontent.com/assets/25882199/23104474/6e27031c-f6c6-11e6-8b44-f20c332e0644.png)

**Comment by bommels** [comment]
Try to run V0.2 and create a new issue if you run into a problem!

**Comment by F2UME** [comment]
Hi bommels, i've just updated and setup etc, i've setup using your yzyio test hmac page, shouldnt it be picking up the hmac? 
![notepad _2017-02-19_18-11-16](https://cloud.githubusercontent.com/assets/25882199/23104929/ef7ccdea-f6ce-11e6-84f3-77e0b1ee2ca1.png)

- I feel you should make the hmac cookie on the test page actually do something when transfered to the chrome session so we know if it's actually done something because mine has just done this and it's a blank page. http://i.imgur.com/Lglx3Bj.png

**Comment by bommels** [comment]
@EternalYZY Try setting the time-out in bot.py line 88 higher

**Comment by F2UME** [comment]
@bommels Yeah, that worked great, i'm just trying to give as much feedback and errors as possible. Great work. Now it opened Chrome etc and threw out these errors. 
![explorer_2017-02-19_18-22-45](https://cloud.githubusercontent.com/assets/25882199/23105040/a3da1bc0-f6d0-11e6-92df-7a1072aa73df.png)
Also "I feel you should make the hmac cookie on the test page actually do something when transfered to the chrome session so we know if it's actually grabbed something because mine has just opened chrome session and is jsut a blank page"

**Comment by bommels** [comment]
Can you try an other selenium version?

**Comment by EternalYZY** [comment]
@bommels How would I go about doing this, the solenium I installed was from requirements.txt using pip?

**Comment by bommels** [comment]
no the chromedriver.exe file, (my bad)

**Comment by F2UME** [comment]
@bommels I was going to say haha, Anyway...

I've changed from ChromeDriver version 2.9 to version 2.8 and 2.7 and still having errors, i'm getting these errors. 
![explorer_2017-02-19_19-29-47](https://cloud.githubusercontent.com/assets/25882199/23105632/12121ef4-f6da-11e6-965c-bd6950e5f9d8.png)

**Comment by F2UME** [comment]
Ok so, i've managed to stop it 'spamming' chrome windows , but after opening Chrome to transfer the session, it goes from > http://i.imgur.com/paqzipq.png then it crashes and says > http://i.imgur.com/eIQbXcL.png So the issue is actually transfering the session, I think the error code is only because the chrome has to be close/forced closed.

**Comment by bommels** [comment]
The issue is chromedriver, for some reason, doesn't work on your machine. What version are you currently using?

**Comment by F2UME** [comment]
@bommels I'm using the newest, 2.27. I don't see how it's my machine when someone has opened a issue for the same problem

**Comment by F2UME** [comment]
Just got on the Pc, changed nothing but decided to give it a try and BOOM! magic! 
![yynj0st](https://cloud.githubusercontent.com/assets/25882199/23122292/892e5e7c-f75b-11e6-934c-26e93f1ee5cb.png) THANKS

----

If the proxy needed a username and password, when the chrome window opens, will a popup appear asking for a log on?

**Comment by bommels** [comment]
Nice, yes it will ask for a login. That login is displayed in the output in cmd (currently says None)
